### PR TITLE
Add a function to delete namespaces starting with 'ns-'

### PR DESCRIPTION
**Pull request by sankarpn**
- Added a function to delete namespace artifact with names starting with 'ns-'

https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-model-in-image-tests/907/consoleFull

### CODE REVIEW

**Comment by rjeberhard** [comment]
@bhavaniravichandran, click approve please in addition to the "lgtm" comment.

File: src/integration-tests/bash/cleanup.sh [hunk 1]
```diff
@@ -685,6 +689,8 @@ genericDelete "$g_arg1" "$g_arg2" "$g_arg3" -friendlyDelete
 genericDelete "$g_arg1" "$g_arg2" "$g_arg3" -forceDelete
 SUCCESS="$?"
 
+deleteNamespacesIT
+
```
Comment (tbarnes-us) [inline src/integration-tests/bash/cleanup.sh:693]: Unlike every other delete, this approach is done in foreground and has no retries.

Let me see if there's an easy alternative.

Comment (tbarnes-us) [reply src/integration-tests/bash/cleanup.sh:693]: Also, it is not specifying the same timeout/force parameters we use everywhere else.

Comment (tbarnes-us) [reply src/integration-tests/bash/cleanup.sh:693]: Finally, it is not affecting the return code for the script.

I'll see if I can come up with something more in-line with the spirit of the rest of the script.

**Comment by sankarpn** [comment]
This changes in this PR is covered by 
https://github.com/oracle/weblogic-kubernetes-operator/pull/1733 
hence closing it.

**Pull request closed without merging**